RELEASE/scripts/autoscend/auto_providers.ash

```diff
@@ -530,6 +530,15 @@ float provideInitiative(int amt, location loc, boolean doEquips, boolean specula
 			return result();
 	}
 
+	if(doEquips && auto_haveCCSC() && have_effect($effect[Peppermint Rush]) == 0 && !get_property("_candyCaneSwordLyle").to_boolean()) {
+		if (!speculative) {
+			string temp = visit_url("place.php?whichplace=monorail&action=monorail_lyle");

```

Comment:
does the ccsc need to be equipped before going to the URL?

Reply:
yep!

Reply:
is it alright to just equip it before and then the method equips the actual init outfit, or do I need some other handling?

Reply:
perfect!